subsys/bluetooth/host/hci_core.c

```diff
@@ -8129,24 +8128,25 @@ int bt_le_adv_start(const struct bt_le_adv_param *param,
 		    const struct bt_data *ad, size_t ad_len,
 		    const struct bt_data *sd, size_t sd_len)
 {
-	if (IS_ENABLED(CONFIG_BT_EXT_ADV) &&
-	    BT_FEAT_LE_EXT_ADV(bt_dev.le.features)) {
-		struct bt_le_ext_adv *adv = adv_new_legacy();
-		int err;
+	struct bt_le_ext_adv *adv = adv_new_legacy();
+	int err;
 
-		if (!adv) {
-			return -ENOMEM;
-		}
+	if (!adv) {
+		return -ENOMEM;
+	}
 
+	if (IS_ENABLED(CONFIG_BT_EXT_ADV) &&
+			BT_FEAT_LE_EXT_ADV(bt_dev.le.features)) {
```

Comment:
Please fix the indentation here, should be 1 tab + 4 spaces, not 3 tabs.

Reply:
Got